### ISSUE TITLE
`rvm get stable` succeeds while offline.

### ISSUE THREAD

**Issue by shalecraig** [rvm/rvm#3372]
Maybe I don't understand what `rvm get stable` does, but I expected a hard failure if I'm offline.

Steps to reproduce:
1. Go offline.
2. run `rvm get stable`
3. `Downloading https://get.rvm.io` is printed to output.

Expected behaviour:
1. A message such as `cannot reach get.rvm.io` is printed

AND one of:
1. `rvm get stable` fails.
OR
2. A message such as `continuing, despite failure` is printed.

Actual Behaviour:
1. `RVM reloaded!` is printed.
2. `rvm get stable` succeeds.

Please let me know if you need any more information.

**Comment by havenwood** [comment]
What's your current version of RVM? I haven't yet been able to reproduce this with rvm 1.26.11. I'm seeing an exit status error code 6 with the warning in red upon failure:
```sh
$ rvm get stable
Could not download rvm-installer, get some help at #rvm IRC channel at freenode servers.
```
I'm wondering if this was possibly fixed after your current version or are you using latest? If not latest could you try with latest? If it's not fixed maybe post the output of `rvm debug`.

**Comment by shalecraig** [comment]
After realizing that I was offline, I hopped online and updated. The newer version I now have doesn't have this bug anymore. I'm going to close this issue.

The [`Could not download rvm-installer`](https://github.com/rvm/rvm/blame/92b907537431e9214f5276bbafa2ba1b219dcacf/scripts/functions/cli#L288) looks like it was added by @mpapis in Oct 2014. I'm sure that the rvm version I was using was from before that, which would explain the problem.
